### ISSUE TITLE
Support more Qt QPA platforms

### ISSUE THREAD

**Issue by iphydf** [probonopd/go-appimage#322]
https://github.com/probonopd/go-appimage/blob/fced8b8831039daa246ab355f4e2335074abc206/src/appimagetool/appdirtool.go#L1284

This code seems to only support XCB. I'd like to deploy a binary that runs with native wayland support using the libqwayland.so platform integration plugin.

**Comment by dawagner** [comment]
Same here. It seems to me the fix is straightforward... maybe I'll give it a try some time. In the meantime, I guess we can work around this by copying the plugin manually.

**Comment by iphydf** [comment]
Yeah I've copied the plugins manually, that works for us for now.

**Comment by AdelKS** [comment]
I understand, thanks for chiming in !
